miio/viomivacuum.py

```diff
@@ -174,19 +174,15 @@ class ViomiMopMode(Enum):
 
 class ViomiVacuumStatus:
     def __init__(self, data):
-        # ["run_state","mode","err_state","battary_life","box_type","mop_type","s_time","s_area",
```

Comment:
Please leave the example query/response in place (simply update it with the new values).

Reply:
fixed

Reply:
Please use the new keys & values so that anyone who is looking for the newly introduced properties has an idea how a response looks like.